blog.html

```diff
@@ -0,0 +1,65 @@
+        <p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+      </div>
+
+      <div class="wdog">
+        <h3>How to Walk Multiple Dogs</h3>
+        <img src="images/blog-2.jpg" alt="4 dogs seating">
+        <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.</p>
+        <p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+      </div>
+
+      <div class="fdog">
+        <h3>Teach Your Dog To Fetch</h3>
+        <img src="images/blog-3.jpg" alt="a girl fetching with her dog">
+        <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.</p>
+        <p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+      </div>
+
+    </div>
+
+    <footer>
+      <h3>ADOPTADOG</h3>
+      <p>Aborpos nisto inci tet aut que volupti beruptiberro to ex <br> esseque invenienihil imaximu scideru ptistiusam consequunt</p>
+      <ul>
+        <li><a href="index.html">Home / </a></li>
+        <li><a href="dogs.html">Dogs / </a></li>
+        <li><a href="blog.html">Blog / </a></li>
+        <li><a href="checkout.html">Checkout</a></li>
+      </ul>
+      <p>© 2018 Adoptadog, LLC | Seattle, WA</p>
```

Comment:
another way to use the copyright sympbol is the `&copy;` instead of the symbol.  It would look like this.

`<p>&copy; 2018 Adoptadog, LLC | Seattle, WA</p>